modules/survey_module/ajax/survey_api.php

```diff
@@ -32,7 +32,7 @@ class DirectDataEntryMainPage
      * Initialize all of the class variables and things required from the
      * REQUEST.
      *
-     * @return none, but as a side-effect modifies class
+     * @return null
```

Comment:
"null" means it returns the value null. I'm pretty sure you're looking for "void"